test/napi/lib/hello.js

```diff
@@ -6,4 +6,12 @@ describe('hello', function() {
     assert.equal(addon.greeting, "Hello, World!");
     assert.equal(addon.greeting, addon.greetingCopy);
   });
+
+  it('should export global singletons for JS primitives', function () {
+    assert.equal(addon.undefined, undefined);
+    assert.ok(addon.hasOwnProperty('undefined'));
+    assert.equal(addon.null, null);
+    assert.equal(addon.true, true);
+    assert.equal(addon.false, false);
```

Comment:
Sorry that I missed this before, but these should likely be `strictEqual`.

Reply:
ah yes :) i should've looked at the chai docs instead of just copying the lines above!